crates/bevy_ecs/src/change_detection.rs

```diff
@@ -32,7 +28,10 @@ pub trait DetectChanges {
     /// Returns true if (and only if) this value been changed since the last execution of this
```

Comment:
This comment is bad. "Has been mutably accessed" is more true.

Reply:
done :)